beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -414,7 +419,10 @@ fn context_bytes<E: EthSpec>(
                         }
                     };
                 }
-                RPCResponse::BlobsByRange(_) | RPCResponse::BlobsByRoot(_) => {
+                RPCResponse::BlobsByRange(_)
+                | RPCResponse::BlobsByRoot(_)
+                | RPCResponse::DataColumnsByRoot(_)
+                | RPCResponse::DataColumnsByRange(_) => {
                     return fork_context.to_context_bytes(ForkName::Deneb);
```

Comment:
Can we add a `TODO(das)` here so that we don't forget to update this when we rebase on Electra?
Or alternatively we could compare `spec.eip7594_fork_epoch` against `deneb_fork_epoch` and `electra_fork_epoch`, and set the fork accordingly.